numpy/linalg/umath_linalg.c.src

```diff
@@ -1858,6 +1864,7 @@ typedef struct geev_params_struct {
     char JOBVR;
 } GEEV_PARAMS_t;
 
+#if 0
```

Comment:
I guess these are obviously just there to help debugging. Maybe create a local `#define`, to make it a bit more obvious (and easier to toggle both)?